cmd/nri-prometheus/config.go

```diff
@@ -83,6 +83,7 @@ func setViperDefaults(viper *viper.Viper) {
 	viper.SetDefault("standalone", true)
 	viper.SetDefault("disable_autodiscovery", false)
 	viper.SetDefault("percentiles", []float64{50.0, 95.0, 99.0})
+	viper.SetDefault("worker_threads", 4)
```

Comment:
why not to increase the default value , since this was causing issues in large environments?

Reply:
it's usually enough. the problem is when the target takes more time to respond it blocks everything.